---
# jekyll prompt
---
<!DOCTYPE html>
<html lang="en-US">
<head>
  <title>Clinical Genomics Implementation Guidance</title>
  <meta charset="UTF-8">
  <link rel="stylesheet" href="fhir.css"/>
  <style type="text/css">
  </style>
</head>
<body>

  <h1 style="text-align:center;">FHIR Genomics Implementation Guidance</h1>

  <h1>Table of Contents</h1>

  <p>
    <ol>
      <li><a href="#backgroud">Background</a></li>
      <li><a href="#overview">Overview</a></li>
      <li><a href="#sequence">Sequence Resource</a></li>
      <li><a href="#observation-genetics">Observation-genetics Profile</a></li>
      <li><a href="#diagnosticreport-genetics">DiagnosticReport-genetics Profile</a></li>
      <li><a href="#diagnosticrequest-genetics">DiagnosticRequest-genetics Profile</a></li>
      <li><a href="#hla">HLA genotyping results Profile</a></li>
      <li><a href="#resource_vs_profiles">Relationship among Sequence resource and genetics profiles</a></li>
      <li><a href="#dam">HL7 Domain Analysis Model Use Cases</a></li>
      <li><a href="#other_examples">Other Genomics Use Cases</a></li>
      <li><a href="#apps">Clinical Genomics Apps</a></li>
      <li><a href="#history">Appendix 1: From DSTU2 Genomics to STU3</a></li>
    </ol>
  </p>

  <h1 id="background">Background</h1>
  <p>
    Rapid advances in sequencing technologies, clinical genetics tests for whole
    genome and exome sequencing are allowing sophisticated genetics testing to
    be used by providers and patients in clinical decisions. Results from such
    tests are used to identify distinct genetic variants that may contribute to
    syndromes, conditions and/or predictive responses to treatments. The
    implementation of precision medicine will depend upon having such data to
    diagnose patients, choose medications, and predict the course of disease and
    care, but will require standards and effective user interfaces.
   </p>
  <p>
    A current technical challenge exists in interoperability, or allowing the
    ability to access and share clinical and genetics data. This
    interoperability includes collection, coding, and retrieval to scale. An
    individual's genetic data set is large, complex and requires curation.
    Unfortunately, incompatible systems and nomenclatures are already in use.
    A standards based ontology that could be adopted to integrate both genetic
    data and clinical information systems will be crucial to accelerating the
    integration of precision medicine and to make sense of genetic testing
    results in a complete clinical context.</p>
  <p>One approach for collecting, coding, and retrieving genetics data comes
    from the <a href="http://genomicsandhealth.org">Global Alliance for Genomics and Health</a>
    (<abbr title="Global Alliance for Genomics and Health">GA4GH</abbr>). The
    GA4GH organization has built and is refining an API and data
    model for the exchange of full sequence genomic information across multiple
    research organizations and platforms. The GA4GH focuses on the needs of
    researchers.
  </p>
  <p>
    A second approach is evolving from <a href="http://hl7.org">Health Level 7</a> (<abbr title="Health Level 7">HL7</abbr>) through its standards family called <a href="http://hl7.org/fhir/">Fast Healthcare Interoperability Resources</a> (<abbr title="Fast Healthcare Interoperability Resources">FHIR</abbr>). FHIR focuses on clinical needs and systems and has principally focused on standardizing traditional clinical data of the sort in EMR systems using very modern web technology-based ideas. FHIR is attractive because it is relatively easy to implement because it is comprised of a set of modular components called resources, which can be easily and incrementally assembled into working systems. The clinical requirements for genetics data is, relative to genomics research needs, utilitarian and reductive because it is about distilling and extracting particular genetics data produced by ever more sophisticated testing for use at the point-of-care. This has made FHIR a very functional framework to initiate an interoperable clinical genetics data standardization to which multiple stakeholders have contributed to this guide.
  </p>
  <p>
    Starting in 2014, the HL7 Clinical Genomics Working Group (CGWG) has been drafting the FHIR framework for clinical genetics testing for general adoption. This effort reached its first major milestone on September 23, 2015, when HL7 released its second FHIR Draft Standard for Trial Use (DSTU2). DSTU2 introduced a Standard genetics profile that applies to the FHIR Observation resource. The profile was created according to the FHIR Clinical Genomics Project Scope Statement that authorized its community development. Using this profile, Observation payloads can return genetic testing results in a standardized manner.
  </p>
  <p>
    Extending Observations rather than creating a new, dedicated FHIR genetics data resource is consistent with the FHIR community mandate because it only adds new resources to an existing resource library when there is a compelling case to do so. We provide the case to add a Sequence resource in this document.
  </p>

  <h1 id="overview">Overview</h1>
  <p>
    Given the clinical significance of genetic sequence data and the wealth of information that may be retrieved for an individual's health and with metadata for the betterment of public health, we propose to move beyond FHIR DSTU2 Standard Genetics profile on Observation and allow increased granularity and less ambiguity by creating a new resource to be called <strong>Sequence</strong>. This resource will be used to hold clinically relevant sequence data in a manner that is both efficient and versatile integrating new and as yet undefined types of genomic and other -omics data that will soon be commonly entered into health records for clinical use.   Sequence will be leveraged by other FHIR resources, including Observation.  This is consistent with how all FHIR resources are designed and used.
  </p>
  <p>
    The FHIR Clinical Genomics Working Group (CGWG) began discussing such a dedicated resource in 2014. The September 2014 Informative Ballot (“HL7 Clinical Genomics, Domain Analysis Model: Clinical Sequencing Release 1”) provided guiding use cases, which initially informed development of the initial Standard Genetics profile that is found in FHIR DSTU2. The same use cases also led to a second Project Scope Statement in May 2015 (approved by HL7 DESD/OO/FMG) to develop a Sequence resource (“Develop FHIR sequence resource for Clinical Genomics”). A preliminary effort to address these issues has been explored and published in context of the <a href="http://smarthealthit.org/an-app-platform-for-healthcare/about/">Substitutable Medical Applications and Reusable Technologies</a> (<abbr title="Substitutable Medical Applications and Reusable Technologies">SMART</abbr>) Platforms Project and described in an article (“<a href="http://jamia.oxfordjournals.org/content/early/2015/07/21/jamia.ocv045.long">SMART on FHIR Genomics: Facilitating standardized clinico-genomic apps</a>”).
  </p>
  <p>
    Sequence is designed to hold genetic sequences in blocks relevant to actionable clinical decision-making. Extensions to Sequence address complex cases and can associate it with repositories for retrieving a patient’s full sequence data, such as those defined by GA4GH. Other changes include a suite of genetics profiles for other FHIR resources. In addition, the Observation-genetics profile adds new references so that an Observation can report genetics test results to be integrated into the EHR. There are also new genetics-extension profiles for DiagnosticReport, DiagnosticRequest and FamilyMemberHistory, respectively, to extend them to report genetics results. We have given all of these FHIR genetics profiles the suffix “-genetics” (e.g. “DiagnosticReport-genetics profile”). New profiles on top of DiagnosticReport have been created for reporting HLA genotyping results.
  </p>
  <p>
    On the following pages, we elaborate upon the rationale for the proposed design, introducing in some detail the following resource and profiles:
    <ul>
      <li>A Sequence resource</li>
      <li>An Observation-genetics profile on Observation</li>
      <li>A DiagnosticReport-genetics profile on DiagnosticReport</li>
      <li>A FamilyMemberHistory-genetics profile on FamilyMemberHistory</li>
      <li>A DiagnosticRequest-genetics profile on DiagnosticRequest</li>
      <li>An HLA-genotyping-results profile on DiagnosticReport</li>
    </ul>
  </p>
  <p>
    With these resource and profiles, FHIR can support a large set of clinical use cases (see <a href="#dam">Section 9</a>, <a href="#other_examples">10</a> , and <a href="#apps">11</a>) and is thus positioned to address all emergent -omics use cases, including Next-Generation Sequencing (<abbr title="Next-Generation Sequencing">NGS</abbr>). These tools are simple to implement, will optimize payload sizes, and help developers avoid redundant retrieval of data.  They are fully specified in the <a href="http://hl7-fhir.github.io">Continuous Integration Build of FHIR</a>, and <a href="#history">Appendix 1</a> of this document shows how DSTU 2.0 can be mapped to the new additions to the resource.
  </p>

  <h1 id="sequence">Sequence Resource</h1>

  <h2>Structure Diagram</h2>

  {% include StructureDefinition-Sequence-diff.xhtml %}

  <h2>Description</h2>
  <p>
    The <a href="http://hl7-fhir.github.io/sequence.html">Sequence resource</a> is designed for next-generation sequencing data.  Patients' observed sequences should be represented by recording reference sequence id/string and detected variants.
  </p>

<p>
  Sequence.referenceSeq: Four optional ways are provided to represent reference sequence in Sequence resource:
  <ol>
    <li>Sequence.referenceSeq.referenceSeqId: Reference sequence id in public database;</li>
    <li>Sequence.referenceSeq.referenceSeqString: Reference sequence string; </li>
    <li>Sequence.referenceSeq.referenceSeqPointer: Reference to observed sequence stored in another sequence entity;  </li>
    <li>Sequence.referenceSeq.genomeBuild, Sequence.reference.chromosome: The combination of genome build and chromosome. </li>
  </ol>
</p>
<p>
The window selects a range from the reference sequence (or genome) that is used to define building block of a current sequence (e.g. Sequence resource instance 1).
</p>
<p>
Sequence.quality: Quality scores for bases in the sequence. Is is intended to be compliant with emerging regulatory needs needs (eg: those found at <a href="https://precision.fda.gov">PrecisionFDA</a>).
</p>
<p>
Sequence.variant: This complex element is used for encoding sequence. When the information of reference sequence and variants are provided the observed sequence will be derived.
</p>

<h3>Internal Pointers</h3>
<p>
Sequence.patient: This element points to a Patient identifier to show that this sequence is related to the same patient.
</p>
<p>
Sequence.specimen: A pointer to specimen identifier, if needed.
</p>
<p>
Sequence.device: A pointer to Device identifier which is used for describing sequencing method (such as chip id,  chip manufacturer etc.)
</p>
<p>
Sequence.pointer: A pointer to a Sequence instance for the next sequence block to build a sequence graph.
</p>

<h3>External Pointers</h3>
<p>
Sequence.repository: This complex element is used to provide a clarifying structure, a base URL, and/or relevant IDs when referring to an external repository.
</p>
<p>
GA4GH Repository Example. If the Sequence resource refers to a GA4GH repository for read info, references to a GA4GH full sequence dataset should conform to GA4GH data models and accessed via the GA4GH API. The URL of a GA4GH repository, ids of a GA4GH variant and read group are contained in the Sequence resource. The URL of a GA4GH repository is an api_base of a GA4GH server that could be called for sequence data. The GA4GH variant set is a collection of call sets and the GA4GH call set is a collection of variant calls, typically for one sample. A variant call represents a determination of genotype with respect to that variant.
</p>
<p>
  VariantSet definition: A VariantSet is a collection of variants and variant calls intended to be analyzed together.
</p>
<p>
  CallSet definition: A CallSet is a collection of calls that were generated by the same analysis of the same sample.
</p>
<p>
  A read group is a collection of reads produced by a sequencer. A read group set typically models reads corresponding to one sample, sequenced one way, and aligned one way. The API reference of <a href="https://cloud.google.com/genomics/">Google Genomics</a> is a GA4GH repository built by Google and provides details of the data models, such as the resource representations.
</p>

<h2>Usage Examples</h2>
<p>
We provide an <a href="http://hl7-fhir.github.io/sequence-example.html">example</a> on how to use the Sequence resource.
</p>

{% include Sequence-example-html.xhtml %}

<p>
Source: <a href="Sequence-example.xml.html">XML</a>, <a href="Sequence-example.json.html">JSON</a>, <a href="Sequence-example.ttl.html">Turtle</a>.
</p>

<img src="images/image15.jpg" alt="Sequence Graph" style="width:624px;height:468px;">
<p>The diagram above describes 4 optional ways provided in the Sequence resource to encode sequencing data.</p>

<img src="images/image34.jpg" alt="" style="width:624px;height:468px">

<h1 id="observation-genetics">Observation-genetics Profile</h1>

<h2>Structure Diagram</h2>

{% include StructureDefinition-observationforgenetics-diff.xhtml %}

<h2>Description</h2>

<p>
The <a href="http://hl7-fhir.github.io/observationforgenetics.html">Observation-genetics profile</a> is used to describe a sequence variant.
</p>
<p>
  Some of the attributes of the profile follow:
  <ul>
    <li>The observation-geneticsSequence extension will refer to the Sequence resource for sequence information related to this variant.</li>
    <li>The observation-geneticsInterpretation extension will refer to an Observation instance which contains clinical interpretations for the variant described.</li>
    <li>The code, effective[x], issued, performer, method, specimen elements can be used to describe how the genetic observation (variant and sequence data) is obtained.</li>
    <li>Other extensions are used to describe attributes of this variant such as Genomics Source Class, Amino Acid Change Type, etc.</li>
    <li>The Observation.result element is used to show whether this variant is detected in this Patient.</li>
  </ul>
</p>
<p>
  Additional Observations instance will be created for variant’s further analysis. For example, Observation.component element will be used for knowledge-based interpretations of the sequence variant. Here are some examples for the component.code.
</p>
<br/>
<div dir="ltr">
    <table>
        <colgroup>
            <col width="96"/>
            <col width="192"/>
            <col width="282"/>
        </colgroup>
        <tbody>
            <tr>
                <td>
                    <p dir="ltr">
                        LOINC Code
                    </p>
                </td>
                <td>
                    <p dir="ltr">
                        LOINC Element Name
                    </p>
                </td>
                <td>
                    <p dir="ltr">
                        Comments
                    </p>
                </td>
            </tr>
            <tr>
                <td>
                    <p dir="ltr">
                        51963-7
                    </p>
                </td>
                <td>
                    <p dir="ltr">
                        Medication Assessed
                    </p>
                </td>
                <td>
                    <p dir="ltr">
                        A coded medication accessed in a pharmacogenetic test (recommend RxNorm).
                    </p>
                </td>
            </tr>
            <tr>
                <td>
                    <p dir="ltr">
                        51967-8
                    </p>
                </td>
                <td>
                    <p dir="ltr">
                        Genetic disease assessed
                    </p>
                </td>
                <td>
                    <p dir="ltr">
                        A coded disease that is associated with the region of DNA covered by the genetic test (recommend SNOMED).
                    </p>
                </td>
            </tr>
            <tr>
                <td>
                    <p dir="ltr">
                        53037-8
                    </p>
                </td>
                <td>
                    <p dir="ltr">
                        Genetic Disease Sequence Variant Interpretation
                    </p>
                </td>
                <td>
                    <p dir="ltr">
                        Interpretation of the pathogenicity of the DNA Sequence Variant in the context of the assessed genetic disease.
                    </p>
                </td>
            </tr>
            <tr>
                <td>
                    <p dir="ltr">
                        53040-2
                    </p>
                </td>
                <td>
                    <p dir="ltr">
                        Drug Metabolism Sequence Variant Interpretation
                    </p>
                </td>
                <td>
                    <p dir="ltr">
                        Predicted phenotype for drug efficacy. A sequence variant interpretation value known to allow (responsive) or prevent (resistant) the
                        drug to perform.
                    </p>
                </td>
            </tr>
            <tr>
                <td>
                    <p dir="ltr">
                        51961-1
                    </p>
                </td>
                <td>
                    <p dir="ltr">
                        Drug Efficacy Sequence Variant Interpretation
                    </p>
                </td>
                <td>
                    <p dir="ltr">
                        Predicted phenotype for ability of drug to bind to intended site in order to deliver intended effect. A Sequence Variant interpretation
                        value known to allow (responsive) or prevent (resistant) the drug to perform.
                    </p>
                    <br/>
                </td>
            </tr>
        </tbody>
    </table>
</div>

<p>
  In the meantime, the related element in this Observation instance will point to Observaiton-genetics profile to show these clinical interpretations are further analysis for the variant.
</p>

<h2>Usage Examples</h2>

<h3>An Observation-genetics instance and its two related Observation instances for interpretations</h3>
<p>
  An <a href="http://hl7-fhir.github.io/observation-example-genetics-1.html">example</a> of an Observation-genetics instance which records a variant detected in the patient.
</p>

{% include Observation-example-genetics-1-html.xhtml %}

<p>
Source: <a href="SObservation-example-genetics-1.xml.html">XML</a>, <a href="Observation-example-genetics-1.json.html">JSON</a>, <a href="Observation-example-genetics-1.ttl.html">Turtle</a>.
</p>

<p>
  An <a href="http://hl7-fhir.github.io/observation-example-genetics-2.html">example</a> of an Observation instance which records knowledge-based clinical interpretations for the variant represented in A.
</p>

{% include Observation-example-genetics-2-html.xhtml %}

<p>
Source: <a href="SObservation-example-genetics-2.xml.html">XML</a>, <a href="Observation-example-genetics-2.json.html">JSON</a>, <a href="Observation-example-genetics-2.ttl.html">Turtle</a>.
</p>

<p>
  An <a href="http://hl7-fhir.github.io/observation-example-genetics-3.html">example</a> of an Observation instance which records PCR validation test for the variant in A.
</p>

{% include Observation-example-genetics-3-html.xhtml %}

<p>
Source: <a href="SObservation-example-genetics-3.xml.html">XML</a>, <a href="Observation-example-genetics-3.json.html">JSON</a>, <a href="Observation-example-genetics-3.ttl.html">Turtle</a>.
</p>

<h1 id="diagnosticreport-genetics">DiagnosticReport-genetics Profile</h1>

<h2>Structure Diagram</h2>

{% include StructureDefinition-reportforgenetics-diff.xhtml %}

<p>
This <a href="http://hl7-fhir.github.io/reportforgenetics.html">DiagnosticReport-genetics</a> is built on top of DiagnosticReport. The new profile is used to describe a genetics test report. The result element in DiagnosticReport will refer to the Observation resource that can lead to a bundle of genetic observations. And the element of code, effective[x], issued, performer, request, specimen are be used to describe the details of the genetic test. Extensions about AssessedCondition and FamilyMemberHistory are added. Overall, this profile extends the DiagnosticReport resource to enable reporting of structured genetic test results. In addition, it denotes condition context for genetic testing, which may influence reported variants and interpretations for large genomic testing panels.
</p>
<p>
<ul>
  <li>The DiagnosticReport-genetics profile contains two extensions referring to other resources/profiles and one complex extension.</li>
  <li>AssessedCondition is used to denote condition context for genetic testing, which may influence reported variants and interpretation for large genomic testing panels. It refers to the Condition resource.</li>
  <li>FamilyMemberHistory are significant health events and conditions for one or more persons related to the patient. It refers to the FamilyMemberHistory-Genetic profile.</li>
  <li>Analysis provides knowledge-based interpretation of overall results of tests performed on patients. </li>
</ul>
    Examples for analysis.code:
    <br/>
    <br/>
    <div dir="ltr">
        <table>
            <colgroup>
                <col width="96"/>
                <col width="186"/>
                <col width="294"/>
            </colgroup>
            <tbody>
                <tr>
                    <td>
                        <p dir="ltr">
                            LOINC Code
                        </p>
                    </td>
                    <td>
                        <p dir="ltr">
                            LOINC Element Name
                        </p>
                    </td>
                    <td>
                        <p dir="ltr">
                            Comments
                        </p>
                    </td>
                </tr>
                <tr>
                    <td>
                        <p dir="ltr">
                            51968-6
                        </p>
                    </td>
                    <td>
                        <p dir="ltr">
                            Genetic Disease Analysis Overall Interpretation
                        </p>
                    </td>
                    <td>
                        <p dir="ltr">
                            Interpretation of all identified DNA Sequence variants along with any known clinical information for the benefit of aiding clinicians
                            in understanding the results overall in either the context of diagnosis or increased risk of disease.
                        </p>
                    </td>
                </tr>
                <tr>
                    <td>
                        <p dir="ltr">
                            51964-5
                        </p>
                    </td>
                    <td>
                        <p dir="ltr">
                            Drug Efficacy Analysis Overall Interpretation
                        </p>
                    </td>
                    <td>
                        <p dir="ltr">
                            Overall predicted phenotype for drug efficacy for all DNA Sequence Variants identified in a single case.
                        </p>
                    </td>
                </tr>
                <tr>
                    <td>
                        <p dir="ltr">
                            51971-0
                        </p>
                    </td>
                    <td>
                        <p dir="ltr">
                            Drug metabolism analysis overall interpretation
                        </p>
                    </td>
                    <td>
                        <p dir="ltr">
                            Overall predicted phenotype for drug metabolism for all DNA Sequence Variants identified in a single case.
                        </p>
                    </td>
                </tr>
            </tbody>
        </table>
    </div>
    <br/>
</p>

<h2>Usage Examples</h2>
<p>
An <a href="http://hl7-fhir.github.io/diagnosticreport-genetics-example-2%20with%20familyhistory.html">example</a> of a genetic test report for a patient with FamilyHistory.
</p>

{% include DiagnosticReport-dg1-html.xhtml %}
<p>
  Source: <a href="DiagnosticReport-dg1.xml.html">XML</a>, <a href="DiagnosticReport-dg1.json.html">JSON</a>, <a href="DiagnosticReport-dg1.ttl.html">Turtle</a>.
</p>

<p>
  An <a href="http://hl7-fhir.github.io/diagnosticreport-genetics-comprehensive-bone-marrow-report.html">example</a> of a comprehensive bone marrow report.
</p>

{% include DiagnosticReport-dg2-html.xhtml %}
<p>
Source: <a href="DiagnosticReport-dg2.xml.html">XML</a>, <a href="DiagnosticReport-dg2.json.html">JSON</a>, <a href="DiagnosticReport-dg2.ttl.html">Turtle</a>.
</p>

<h1 id="diagnosticrequest-genetics">DiagnosticRequest-genetics profile</h1>

<h2>Structure Diagram</h2>
<p>
  A complex extension is added on top of the DiagnosticRequest resource. Here is the structure of the extension:
</p>

{% include StructureDefinition-orderforgenetics-diff.xhtml %}

<h2>Description</h2>
<p>
  To describe an order requested sequence variants detection. User must set up the code for the request and they can also refer to the corresponding sequence instance for that variant.
</p>

<h2>Usage Example</h2>
<h3>An order for genetics test</h3>
<p>
  <a href="http://hl7-fhir.github.io/diagnosticrequest-genetics-example-1.html">Here</a> is a diagnostic request for testing 185delAG variant. The mother of the patient received results from a mutation panel (eg. MyRisk from Myriad) and she has a BRCA1 185delAG mutation. The clinician the would like to request to test the patient only for an 185delAG mutation. In this case, the diagnostic request for the patient will specify the sequence variant - 185delAG.
</p>

<h1 id="hla">HLA genotyping results Profile</h1>
<p>
  Human leukocyte antigen (HLA) genotyping is fundamental for research and clinical practice in immunogenetics and histocompatibility. Pointers to external locations refer to registered methods, raw NGS reads, and reference standards can be conveyed in this profile. Information about allele assignment including ambiguous results and the allele database used for assignments is stored in extensions.
</p>
<p>
  The structure of the <a href="http://hl7-fhir.github.io/hlaresult.html">HLA typing report</a> in this profile attempts to follow the principles outlined in the Minimum Information for ReportIng Next-generation sequence Genotyping (MIRING). These principles were identified through a series of meetings with international group of stakeholders in the  application of Next Generation Sequencing (NGS) technology for genotyping the HLA and KIR loci as well as other immune-related loci (http://igdawg.org/ngs.html). MIRING describes eight principles, described in detail in <a href="http://www.ncbi.nlm.nih.gov/pubmed/26319908">Human Immunology. 2015 Dec; 76(12):954-962</a>. These include detailed metadata about:
  <ol>
    <li>MIRING Annotation</li>
    <li>Reference Context</li>
    <li>Full Genotype</li>
    <li>Consensus Sequence</li>
    <li>Novel Polymorphisms</li>
    <li>Platform Documentation</li>
    <li>Read Processing Documentation</li>
    <li>Primary Data</li>
  </ol>
  (see the publication above for details about each)
</p>
<p>
  These principles were implemented in a technical specification by extending an existing XML based format for exchanging histocompatibility and immunogenetic genotyping data called Histoimmunogenetics Markup Language (HML) to include results from NGS methodologies  (https://bioinformatics.bethematchclinical.org/hla-resources/hml/). The resulting schema may be found in https://schemas.nmdp.org/. The National Marrow Donor Program (NMDP)/Be The Match uses this format for reporting HLA genotyping from potential donors and for patients needing stem cell transplants.
</p>
<p>
  Both MIRING and HML were used to inform mapping data elements to FHIR resources such as Patient, Specimen, Sequence, ObservationforGenetics Profile, and DiagnosticReport  where possible. Several additional data elements were needed specifically for this use case, resulting in the development of a specific profile for reporting HLA genotyping results (Diagnostic Report Profile for HLA Genotyping Results).
  <ul>
    <li>Minimum information for reporting next generation sequence genotyping (MIRING): Guidelines for reporting HLA and KIR genotyping via next generation sequencing (see <a href="http://www.ncbi.nlm.nih.gov/pubmed/26407912">here</a>)</li>
    <li>Histoimmunogenetics Markup Language 1.0: Reporting Next Generation Sequencing-based HLA and KIR Genotyping (see <a href="http://www.ncbi.nlm.nih.gov/pubmed/26319908">here</a>)</li>
  </ul>
</p>

<h2>Structure Diagram of HLA genotyping results Profile</h2>

{% include StructureDefinition-hla-genotyping-results-haploid-diff.xhtml %}

<p>
  Both hla-genotyping-results-glstring and hla-genotyping-results-haploid are complex extensions.
</p>
<p>
  The development of these extensions were informed from the allele-assignment structure found in HML. While allele assignment for individual loci can be reported in an Observation, here they are used to summarize the assignments at a report level. Two methods may be used for reporting HLA allele-assignments: haploid and GL String.
</p>
<p>
  Structure of hla-genotyping-results-haploid:
</p>

{% include StructureDefinition-hla-genotyping-results-haploid-diff.xhtml %}

<p>
  Haploid reporting is a method that is used to report an individual allele, or a list of possible alleles if the results are ambiguous, using NMDP Multiple Allele Codes (MAC) which is a shorthand for describing allele ambiguity (https://bioinformatics.bethematchclinical.org/hla-resources/allele-codes/). MACs are widely used for reporting HLA typing results because they are able to report long allele lists in a small amount of space, but the system has a number of shortcomings (<a href="http://www.ncbi.nlm.nih.gov/pubmed/23849068">Tissue Antigens. 2013 Aug;82(2):106-12</a>). When reporting data using haploid, typical use is one or two haploid elements for a particular locus, but possibly more if multiple loci are covered (ex: two HLA-DRB1 haploids + one HLA-DRB3 haploid).
</p>
<p>
  Within Haploid, Method indicates whether the general methodology is DNA based typing (e.g., Sequence Specific Primers (SSP), Sequence Specific Oligonucleotide Probes (SSOP), or Sequence Based Typing (SBT)), or SER which indicates serology based methods.
</p>
<p>
  Example of using two hla-genotyping-results-haploid extensions to report a HLA-A genotype of HLA-A*01:AB and HLA-A*02:MN which expands to HLA-A*01:01/HLA-A*01:02 and HLA-A:02:01/HLA-A:02:02/HLA-A:02:03
</p>

<textarea rows="55" cols="110" style="border:none;">
  <extension url="http://hl7.org/fhir/StructureDefinition/hla-genotyping-results-haploid" >
    <extension url=”locus”>
        <valueCodeableConcept>
            <coding>
                <system value="http://www.genenames.org"/>
                <code value="4931"/>
                <display value="HLA-A"/>
            </coding>
            <text value="HLA-A"/>
        </valueCodeableConcept>
    </extension>
    <extension url=”type”>
        <valueCodeableConcept>
             <coding>
                <system value="https://bioinformatics.bethematchclinical.org/hla-resources/allele-codes/"/>
                <code value="AB"/>
                <display value="01/02"/>
            </coding>
            <text value="HLA-A*01:AB"/>
        </valueCodeableConcept>
    </extension>
    <extension url=”method”>
        <valueCodeableConcept>
            <text value=”DNA”/>
        </valueCodeableConcept>
    </extension>
</extension>
<extension url="http://hl7.org/fhir/StructureDefinition/hla-genotyping-results-haploid" >
    <extension url=”locus”>
        <valueCodeableConcept>
            <coding>
                <system value="http://www.genenames.org"/>
                <code value="4931"/>
                <display value="HLA-A"/>
            </coding>
            <text value="HLA-A"/>
        </valueCodeableConcept>
    </extension>
    <extension url=”type”>
        <valueCodeableConcept>
             <coding>
                <system value="https://bioinformatics.bethematchclinical.org/hla-resources/allele-codes/"/>
                <code value="AB"/>
                <display value="01/02/03"/>
            </coding>
            <text value="HLA-A*02:MN"/>
        </valueCodeableConcept>
    </extension>
    <extension url=”method”>
        <valueCodeableConcept>
            <text value=”DNA”/>
        </valueCodeableConcept>
    </extension>
  </extension>
</textarea>

<p>
  Because of limitations with MAC, another method called GL Strings was developed that encodes the results in a text string with hierarchical set of operators to describe the relationships between alleles, lists of possible alleles, phased alleles, genotypes, lists of possible genotypes, and multilocus unphased genotypes, without losing typing information or increasing typing ambiguity. (<a href="http://www.ncbi.nlm.nih.gov/pubmed/23849068">Tissue Antigens. 2013 Aug;82(2):106-12</a>).
</p>
<p>
  The structure of HLA-genotyping-results-glstring:
</p>

{% include StructureDefinition-hla-genotyping-results-glstring-diff.xhtml %}

<p>
  While the string format is easily parsed into separate components to be rendered for user viewing, GL Strings by themselves are potentially quite long and difficult to read. It often advantageous to point to a URI which may return the GL String on demand to avoid manual data entry. This is available through the URI element in this extension.
</p>
<p>
  Example of using the hla-genotyping-results-glstring to report a GL String in both text and URI formats
</p>

<textarea rows="9" cols="110" style="border:none;">
  <extension url="http://hl7.org/fhir/StructureDefinition/hla-genotyping-results-glstring">
    <extension url=”text”>
        <valueString value="HLA-A*01:01:01:01/HLA-A*01:02+HLA-A*02:01:02/HLA-A*02:02:02/HLA-A*02:03:01"/>
    </extension>
    <extension url=”uri”>
        <valueUri value="https://gl.nmdp.org/imgt-hla/3.23.0/genotype/1h"/>
    </extension>
  </extension>
</textarea>

<p>
  Whether reporting in Haploid or GL String formats, it is important to identify the version of the IMGT/HLA allele database, as new HLA alleles are constantly being discovered and allele assignment is based on the known alleles at the time, and so results may need to be reinterpreted later. This is done through the Allele Database element.
</p>
<p>
  Structure of hla-genotyping-results-allele-database
</p>

{% include StructureDefinition-hla-genotyping-results-allele-database-diff.xhtml %}

<p>
  Example of using hla-genotyping-results-allele-database to report using the IMGT/HLA database, version 3.23
</p>

<textarea rows="10" cols="110" style="border:none;">
  <extension url="http://hl7.org/fhir/StructureDefinition/hla-genotyping-results-allele-database">
    <valueCodeableConcept>
        <coding>
            <system value="http://www.ebi.ac.uk/ipd/imgt/hla/"/>
            <version value="3.23"/>
        </coding>
        <text value="IMGT/HLA 3.23"/>
    </valueCodeableConcept>
  </extension>
</textarea>

<p>
  Overall methodology may be reported using hla-genotyping-results-method. Here a codeable concept may be used to refer to method entered into a public registry, such as the NCBI Genetic Test Registry, or a local private registry.
</p>
<p>
  Structure of hla-genotyping-results-method
</p>

{% include StructureDefinition-hla-genotyping-results-method-diff.xhtml %}

<p>
  Example of hla-genotyping-results-method to report a lab test registered in the NCBI Genetic Test Registry
</p>

<textarea rows="10" cols="110" style="border:none;">
  <extension  uri="http://hl7.org/fhir/StructureDefinition/hla-genotyping-results-method">
    <valueCodeableConcept>
          <coding>
              <system value="http://www.ncbi.nlm.nih.gov/gtr/"/>
              <code value=GTR000000000.0>
          </coding>
          <text value=”NGS based HLA-A genotyping”
    </valueCodeableConcept>
  </extension>
</textarea>

<h2>Usage Example</h2>

<p>
  Example of a HLA genotyping report
  <ul>
    <li>that was requested in a Diagnostic Request,</li>
    <li>with results for HLA-A, HLA-B, HLA-C loci, </li>
    <li>by Sequence Based Typing (SBT) of exons 2 and 3,</li>
    <li>using Next Generation Sequencing technologies,</li>
    <li>allele assignment based on  the IMGT/HLA database version 3.23,</li>
    <li>reported as a multilocus unphased genotype using GL Strings,</li>
    <li>and with genotyping results for individual loci available in separate Observations</li>
  </ul>
</p>

<textarea rows="91" cols="110" style="border:none;">
  <DiagnosticReport>
    <text>
      <div xmlns="http://www.w3.org/1999/xhtml">
        <h3>HLA-A, -B, -C Genotyping Report for Marry Chalmers (MRN: 12345) </h3>
        <pre>
        Gene      Allele 1                  Allele 2
        HLA-A     HLA-A*01:01:01:01         HLA-A*01:02
        HLA-B     HLA-B*15:01:01:01         HLA-B*57:01:01
        HLA-C     HLA-C*01:02:01            HLA-C*03:04:01:01
        </pre>
        <p>Allele assignments based on IMGT/HLA 3.23</p>
        <p>Issued: 2015-05-26T15:30:10+01:00</p>
        <p>Method: Next-Generation Sequencing of exons 2 and 3 of HLA Class I genes</p>
        <p>Lab: ACME labs, Inc</p>
      </div>
    </text>
    <code>
        <coding>
            <system value="http://loinc.org"/>
            <code value="53767-0"/>
            <display value="HLA-A+​B+​C SBT [Interpretation] by High resolution"/>
        </coding>
    </code>
    <category>
        <coding>
            <system value="http://hl7.org/fhir/ValueSet/diagnostic-service-sections"/>
            <code value="GE"/>
            <display value="Genetics"/>
        </coding>
    </category>
    <subject>
        <reference value="Patient/123"/>
        <display value="Mary Chalmers"/>
    </subject>
    <specimen>
        <reference value="Specimen/456"/>
        <display value="buccal swab"/>
    </specimen>
    <request>
        <reference value=”DiagnosticRequest/789”/>
        <display value=”Order for HLA-A,-B,-C genotyping”/>
    </request>
    <effectiveDateTime value="2015-05-26T15:30:10+01:00"/>
    <issued value="2014-05-16T10:28:00+01:00"/>
    <extension xmlns="http://hl7.org/fhir"
         url="http://hl7.org/fhir/StructureDefinition/hla-genotyping-resultsAlleleAsignmentDate">
        <valueDate value="2014-05-16T10:28:00+01:00"/>
    </extension>
    <extension xmlns="http://hl7.org/fhir"
         url="http://hl7.org/fhir/StructureDefinition/hla-genotyping-results-allele-database">
        <valueCodeableConcept>
            <coding>
                <system value="http://www.ebi.ac.uk/ipd/imgt/hla/"/>
                <version value="3.23"/>
            </coding>
            <text value="IMGT/HLA 3.23"/>
        </valueCodeableConcept>
    </extension>
    <extension xmlns="http://hl7.org/fhir"
        url="http://hl7.org/fhir/StructureDefinition/hla-genotyping-results-glstring" >
        <extension url="text">
            <valueString value="HLA-A*01:01:01:01+HLA-A*01:02^HLA-B*15:01:01:01+HLA-B*57:01:01^HLA-C*01:02:01+HLA-C*03:04:01:01"/>
        </extension>
        <extension url="uri">
            <valueUri value="https://gl.nmdp.org/imgt-hla/3.23.0/multilocus-unphased-genotype/i" />
        </extension>
    </extension>
    <extension  xmlns="http://hl7.org/fhir"
        url="http://hl7.org/fhir/StructureDefinition/hla-genotyping-results-method">
        <valueCodeableConcept>
            <coding>
                <system value="http://www.ncbi.nlm.nih.gov/gtr/"/>
                <code value="GTR000000000.0"/>
            </coding>
            <text value="Next Generation Sequencing of exons 2 and 3 of HLA Class I genes"/>
        </valueCodeableConcept>
    </extension>
    <result>
        <reference value="Observation/111"/>
        <display value="Genotype Observation for for HLA-A"/>
    </result>
    <result>
        <reference value="Observation/222"/>
        <display value="Genotype Observation for for HLA-B"/>
    </result>
    <result>
        <reference value="Observation/333"/>
        <display value="Genotype Observation for for HLA-C"/>
    </result>
  </DiagnosticReport>
</textarea>

<p>
  In the above example, three Observations are referenced, each containing the details surrounding the genotyping of each gene/locus. Those observations would further refer to other observations (using Observation.related) that point to phased exon sequences which are the basis for individual allele assignment. This strategy is illustrated in the following figure for one gene, and sequencing two exons for each allele. Note that references from DiagnosticReport and Observation back to Patient/Donor and Specimen are not shown for clarity, although we would include them in the resource instances.
</p>

<img src="images/image25.png" alt="Phasing" style="width:624px;height:396px;">

<h1 id="resource_vs_profiles">Relationship among Sequence resource and genetics profiles</h1>

<img alt="" src="images/image00.png" style="width: 624.00px; height: 428.00px;">

<h2>Pointers to Sequence resource</h2>

<p>
  Sequence resource is used to transmit and represent sequencing data. In FHIR Genomics, there are several genetics profiles containing internal pointers to Sequence resource for genetic data reporting. Here is the summary:
</p>
<p>
  Sequence.pointer: A bundle of Sequence instances can be connected by Sequence.pointer to build sequence graph described in Section 3. Sequence.pointer will point to the next sequence block.
</p>
<p>
  Sequence extension in Observation-genetics profile: Observation-genetics profile is used to report a genetic variant found in patients. Sequence extension contains a pointer to Sequence identifier which has related sequencing read info. It will provide reference/observed allele information and quality scores for each based/sequence block.
</p>

<h2>Pointer to Observation resource (based on genetics profile)</h2>

<p>
  Observation-genetics based Observation resource is used for interpretative genetic data.  Sequence resource and genetics profiles will use internal pointers to Observation-genetics-profile based Observation instance for variant report.
</p>
<p>
  Sequence.observation: A pointer to genetics Observation instance which is used for interpretations of this sequence block. Interpretations are generally about genetic variant found is this sequence block.
</p>
<p>
  DiagnosticReport.result: A pointer to Observation instance. This element is used for reporting genetic result.
</p>
<p>
  Item extension in DiagnosticRequest-genetics profile: It is used to describe the genetic test order.
</p>
<p>
  FamilyMemberHistory:  A pointer to genetics Observation instance to report genetic test results of family member
</p>

<h2>Observation vs. Sequence</h2>

<p>
  Sequence is used for raw sequencing data while genetic-profile-based Observation resource is used for reporting interpretative genetic information (eg. DNA/amino acid variant detected in the patient). Although both Sequence and Observation-genetics profiles contain variant info, they are used differently: Sequence.variant is used for encoding the sequence block; the Observation-genetics profile is mainly used for a variants test result or interpretations of raw data in Sequence resource.
</p>

<h2>Usage example</h2>

<img alt="" src="images/image09.png" style="width: 624.00px; height: 457.33px;">

<p>
  This use case depicts the whole story of FHIR in the clinical genomics setting: an order is requested, then test results are reported. A Sequence instance and two Observation instances for further interpretation are also created.
</p>

<h1 id="dam">HL7 Domain Analysis Use Cases</h1>

<p>
  Simulated use of the proposed Sequence resource and profiles on other FHIR resources are shown for many of the examples in Section 5 – Use Case Scenarios in the <a href="https://www.hl7.org/documentcenter/public_temp_649E1787-1C23-BA17-0CA4C354EB3E41D5/wg/clingenomics/docs/V3DAM_CG_CLINSEQ_R1_O1_2013JAN.pdf">HL7 Domain Analysis Model (DAM): Clinical Genomics, Release 1, September 2014 Informative Ballot</a>.
</p>

<p>
  We show use of the FHIR design we are proposing for these cases:
  <ol>
    <li>Specimen Identification
      <ol>
        <li>Get references to all variants obtained from germline analysis.</li>
        <li>Explore antibacterial drug resistance over time (non-patient DNA)</li>
      </ol>
    </li>
    <li>Clinical Sequencing – Germline Testing</li>
    <li>Cancer Profiling</li>
    <li>Decision Making Tools – Family History and Drug Dosage Calculators
      <ol>
        <li>Today clinicians translate (i.e. manually re-enter) genetic data into tools for decision making. This includes family history tools and drug dosage calculators. In the future, this data will automatically be incorporated into clinical decision making tools.</li>
        <li>Get pharmacogenomic reporting and disease risk or diagnosis associated with variant</li>
      </ol>
    </li>
    <li>Public Health</li>
    <li>Clinical and Research Data Warehouses</li>
  </ol>
</p>

<h2>Specimen Identification</h2>

<ul>
  <li>Germline testing for biomarkers/mutations (usually inherited)</li>
  <li>Tumor testing for biomarkers/mutations (somatic / tumor specific)</li>
  <li>Pediatric Testing</li>
  <li>Infectious Disease Testing</li>
</ul>

<h3>Get references to all variants obtained from germline analysis</h3>

<textarea rows="4" cols="110" style="border:none;">
  GET   /Observation?
                _profile=http://hl7.org/fhir/StructureDefinition/observationforgenetics&
                GenomicSourceClass=http://hl7.org/fhir/LOINC-48002-0-answerlist|LA6683-2
</textarea>

<br />

<img alt="" src="images/image30.png" style="width: 624.00px; height: 362.67px;">

<p>
  *Return a bundle of genetics-profile-based Observation instances
</p>

<h2>Clinical Sequencing – Germline Testing</h2>

<p>
  For a chart review, get references to all DNA sequences related to mutations with an interpretation “Unknown Significance”
</p>

<textarea rows="4" cols="110" style="border:none;">
  GET /Observation?
subject=123&
component-code-value-[x]= http://loinc.org|53037-8$LA6682-4
</textarea>

<p>
  53037-8: LOINC code for “Genetic disease sequence variant interpretation”
</p>
<p>
  LA6682-4: LOINC answer code for “Unknown Significance”.
</p>

<img alt="" src="images/image04.png" style="width: 624.00px; height: 461.33px;">

<h2>Cancer Profiling</h2>

<p>
  The goal of this profile methodology is to get references from all variants obtained from somatic analysis. Changes in the population of cells with particular mutations will change overtime as well as in conjunction with events such as therapy. For instance, targeted chemotherapy may kill a specific population of cancer cells with specific mutations and other cancer cell populations may survive and continue to divide. Therefore, clearly annotating these specimens as somatic variants and capturing annotations related to a time relevant to a treatment timeline may be critical for analysis.
</p>

<textarea rows="6" cols="110" style="border:none;">
  GET /Observation?
              _profile=http://hl7.org/fhir/StructureDefinition/observationforgenetics&
GenomicsSourceClass=http://hl7.org/fhir/LOINC-48002-0-answerlist|LA6684-0&
date=2015-07-04&
subject=123
</textarea>

<br />

<img alt="" src="images/image37.png" style="width: 624.00px; height: 400.00px;">

<h2>Decision Making Tools – Family History and Drug Dosage Calculators</h2>

<p>
  Today clinicians translate (i.e. manually re-enter) genetic data into tools for decision making. This includes family history tools and drug dosage calculators. In the future, this data will automatically be incorporated into clinical decision making tools.
</p>

<p>
  Get family history related to one observation:
</p>

<textarea rows="4" cols="110" style="border:none;">
  GET /DiagnosticReport?
              _profile=http://hl7.org/fhir/StructureDefinition/reportforgenetics&
	subject=123
</textarea>

<br />

<img alt="" src="images/image02.png" style="width: 604.00px; height: 309.00px;">

<h3>Get pharmacogenomic reporting and disease risk or diagnosis associated with variant</h3>

<textarea rows="5" cols="110" style="border:none;">
  GET /Observation?
              		_profile=http://hl7.org/fhir/StructureDefinition/observationforgenetics&
subject=123&
Interpretation.component-code= http://loinc.org|51963-7
</textarea>

<p>
  51963-7 : LOINC code for “Medication Assessed ”
</p>

<img alt="" src="images/image08.png" style="width: 624.00px; height: 480.00px;">

<h2>Public Health</h2>

<p>
  Today Registrars manually translate clinical data into public health reporting systems. This data is used to monitor and improve public health (e.g. surveillance and clinical research). In the future, this data will be extracted from the EHR in an automated (or semi-automated) fashion.
</p>

<p>
  For a breast cancer clinical genomic study, get all genetic-profile-based observations of patients with breast cancer:
</p>

<textarea rows="4" cols="110" style="border:none;">
  GET /DiagnosticReport?
              _profile=http://hl7.org/fhir/StructureDefinition/reportforgenetics&
	AssessedCondition.code=http://snomed.info/sct|254837009
</textarea>

<br />

<img alt="" src="images/image29.png" style="width: 620.00px; height: 444.00px;">

<h2>Clinical and Research Data Warehouses</h2>

<p>
  Health data warehousing should persist data in its standardized formats, while allowing users to export subsets of the data in the warehouse into multiple ‘data marts’, optimized for specific use cases, analysis type or reporting needs.
</p>
<p>
  Get all genetic-profile-based observations of patients with the variant c.181T>G
</p>

<textarea rows="4" cols="110" style="border:none;">
  GET /Observation?
_profile=http://hl7.org/fhir/StructureDefinition/observationforgenetics&
DNAVariantID=http://www.ncbi.nlm.nih.gov/projects/SNP|rs58238560
</textarea>

<br />

<img alt="" src="images/image07.png" style="width: 624.00px; height: 364.00px;">

<h1 id="other_examples">Other Genomics Use Cases</h1>

<p>
  During the development of the FHIR Genomics design, CGWG participants have commented on the earlier use cases and/or proposed new use cases. In this section, we list describe of these new cases and demonstrate how the proposed design will address them.
</p>

<h2>Get a patient’s family members genetics report</h2>

<p>
  This example is proposed by Kevin Hughes. Family history is useful for clinicians to know more about the condition of the patient.
</p>

<textarea rows="4" cols="110" style="border:none;">
  Get /FamilyMemberHistory?
_profile=http://hl7.org/fhir/StructureDefinition/familymemberhistory-genetic&
patient=123
</textarea>

<br />

<img alt="" src="images/image22.png" style="width: 624.00px; height: 292.00px;">

<h2>Panel</h2>

<p>
  Search for results from nephrotic syndrome panel
</p>

<textarea rows="5" cols="110" style="border:none;">
  Get /DiagnosticReport&
_profile=http://hl7.org/fhir/StructureDefinition/reportforgenetics&
code=N0336&
patient=123
</textarea>

<br />

<img alt="" src="images/image13.png" style="width: 485.50px; height: 248.20px;">

<h2>Sequence Test Metrics</h2>

<h1 id="apps">Clinical Genomic Apps</h1>

<p>
  The idea for a Sequence resource grew out, in part, the SMART Platforms Project, which explored creating clinical genomic apps to integrated traditional EMR clinical data and genomic data to show data visualization and analysis, including CDS that depended upon both types of data.  Below are a couple of examples.  Several apps have already been designed including Genomics Advisor, SMART Precision Cancer Medicine, and Diabetes Bear EMR.  Below, one of these apps will be described.  To include other apps in this section, please feel free to add a note on it and how it uses FHIR/Genomics calls.
</p>

<h2>Genomics Advisor</h2>

<p>
  The SMART on FHIR Genomics Advisor was an app incorporating genomics data to show risk of disease, drug susceptibility, and related conditions based upon genotype. Technically, this app was architected (see below) by combining data from independent data services, a SMART on FHIR clinical server for clinical information and one for a SMART on FHIR Genomics data server for genomic data.  The set of FHIR API calls that are necessary to support this app are shown below:
</p>

<img alt="" src="images/image26.png" style="width: 503.49px; height: 240.48px;">

<p>
  The presentation of the app looks like this:
</p>

<img alt="" src="images/image31.png" style="width: 542.46px; height: 312.63px;">

<h1 id="history">Appendix 1: From DSTU2 Genomics to STU3</h1>

<p>
  The table shows how the data elements in the DSTU2 Observation resource as extended by the Standard Genetics Profile would be mapped to the new Sequence resource.
</p>

<table>
    <colgroup>
        <col width="184"/>
        <col width="213"/>
        <col width="203"/>
    </colgroup>
    <tbody>
        <tr>
            <td>
                <p dir="ltr">
                    DSTU2 Standard Genetics profile
                </p>
                <p dir="ltr">
                    on Observation
                </p>
            </td>
            <td>
                <p dir="ltr">
                    Observation-genetics profile
                </p>
            </td>
            <td>
                <p dir="ltr">
                    Maps to Sequence (Proposed)
                </p>
            </td>
        </tr>
        <tr>
            <td>
                <p dir="ltr">
                    geneticsGenomeBuild
                </p>
            </td>
            <td>
                <p dir="ltr">
                    Sequence extension
                </p>
            </td>
            <td>
                <p dir="ltr">
                    referenceSeq.genomicsBuild
                </p>
            </td>
        </tr>
        <tr>
            <td>
                <p dir="ltr">
                    geneticsChromosome
                </p>
            </td>
            <td>
                <p dir="ltr">
                    Sequence extension
                </p>
            </td>
            <td>
                <p dir="ltr">
                    referenceSeq.chromosome
                </p>
            </td>
        </tr>
        <tr>
            <td>
                <p dir="ltr">
                    geneticsGenomicsStart
                </p>
            </td>
            <td>
                <p dir="ltr">
                    Sequence extension
                </p>
            </td>
            <td>
                <p dir="ltr">
                    variant.start
                </p>
            </td>
        </tr>
        <tr>
            <td>
                <p dir="ltr">
                    geneticsGenomicsStop
                </p>
            </td>
            <td>
                <p dir="ltr">
                    Sequence extension
                </p>
            </td>
            <td>
                <p dir="ltr">
                    variant.end
                </p>
            </td>
        </tr>
        <tr>
            <td>
                <p dir="ltr">
                    geneticsReferenceAllele
                </p>
            </td>
            <td>
                <p dir="ltr">
                    Sequence extension
                </p>
            </td>
            <td>
                <p dir="ltr">
                    variant.referenceAllele
                </p>
            </td>
        </tr>
        <tr>
            <td>
                <p dir="ltr">
                    geneticsObservedAllele
                </p>
            </td>
            <td>
                <p dir="ltr">
                    Sequence extension
                </p>
            </td>
            <td>
                <p dir="ltr">
                    variant.observedAllele
                </p>
            </td>
        </tr>
        <tr>
            <td>
                <p dir="ltr">
                    geneticsSpecies
                </p>
            </td>
            <td>
                <p dir="ltr">
                    Sequence extension
                </p>
            </td>
            <td>
                <p dir="ltr">
                    specimen
                </p>
            </td>
        </tr>
        <tr>
            <td>
                <p dir="ltr">
                    geneticsAllelicState
                </p>
            </td>
            <td>
                <p dir="ltr">
                    Allelic State extension
                </p>
            </td>
            <td>
                <br/>
            </td>
        </tr>
        <tr>
            <td>
                <p dir="ltr">
                    geneticsAllelicFrequency
                </p>
            </td>
            <td>
                <p dir="ltr">
                    Allelic Frequency extension
                </p>
            </td>
            <td>
                <br/>
            </td>
        </tr>
        <tr>
            <td>
                <p dir="ltr">
                    geneticsReadCoverage
                </p>
            </td>
            <td>
                <p dir="ltr">
                    Sequence extension
                </p>
            </td>
            <td>
                <p dir="ltr">
                    readCoverage
                </p>
            </td>
        </tr>
        <tr>
            <td>
                <p dir="ltr">
                    geneticsCopyNumberEvent
                </p>
            </td>
            <td>
                <p dir="ltr">
                    Copy number event extension
                </p>
            </td>
            <td>
                <br/>
            </td>
        </tr>
        <tr>
            <td>
                <p dir="ltr">
                    geneticsTranscriptReferenceSequenceId
                </p>
            </td>
            <td>
                <p dir="ltr">
                    Transcript reference sequence identifier extension
                </p>
            </td>
            <td>
                <br/>
            </td>
        </tr>
        <tr>
            <td>
                <p dir="ltr">
                    geneticsProteinReferenceSequenceId
                </p>
            </td>
            <td>
                <p dir="ltr">
                    Protein reference sequence identifier
                </p>
            </td>
            <td>
                <br/>
            </td>
        </tr>
        <tr>
            <td>
                <p dir="ltr">
                    geneticsCIGAR
                </p>
            </td>
            <td>
                <p dir="ltr">
                    Sequence extension
                </p>
            </td>
            <td>
                <p dir="ltr">
                    variant.cigar
                </p>
            </td>
        </tr>
        <tr>
            <td>
                <p dir="ltr">
                    geneticsVariationId
                </p>
            </td>
            <td>
                <p dir="ltr">
                    DNA variation id extension
                </p>
            </td>
            <td>
                <br/>
            </td>
        </tr>
        <tr>
            <td>
                <p dir="ltr">
                    geneticsVariationType
                </p>
            </td>
            <td>
                <p dir="ltr">
                    DNA sequence variation type extension
                </p>
            </td>
            <td>
                <br/>
            </td>
        </tr>
        <tr>
            <td>
                <p dir="ltr">
                    geneticsAminoAcidChange
                </p>
            </td>
            <td>
                <p dir="ltr">
                    Amino acid change extension
                </p>
            </td>
            <td>
                <br/>
            </td>
        </tr>
        <tr>
            <td>
                <p dir="ltr">
                    geneticsAminoAcidChangeType
                </p>
            </td>
            <td>
                <p dir="ltr">
                    Amino acid change type extension
                </p>
            </td>
            <td>
                <br/>
            </td>
        </tr>
        <tr>
            <td>
                <p dir="ltr">
                    geneticsGene
                </p>
            </td>
            <td>
                <p dir="ltr">
                    Gene extension
                </p>
            </td>
            <td>
                <br/>
            </td>
        </tr>
        <tr>
            <td>
                <p dir="ltr">
                    geneticsDNARegionName
                </p>
            </td>
            <td>
                <p dir="ltr">
                    DNA region name extension
                </p>
            </td>
            <td>
                <br/>
            </td>
        </tr>
        <tr>
            <td>
                <p dir="ltr">
                    geneticsGenomicSourceClass
                </p>
            </td>
            <td>
                <p dir="ltr">
                    Genomic source class extension
                </p>
            </td>
            <td>
                <br/>
            </td>
        </tr>
    </tbody>
</table>


</body>
</html>
